### ISSUE TITLE
Adding immortality privs at 5 million XP 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#6]
Hedgehog: 
Immortality would be that you won't die after you take a deep fall, you can't get killed, etc.  
Your HP will go to zero, and you will always have your hunger pangs, but being killed by players or mobsters, or going into space without a space suit you won't end up with finding out where your bones are this time.

SX:
Other things probably too over powered but for immortality some item you must hold on hand to gain temporary immortality could work allowing use after 750k XP... well not sure if that's good way of mixing XP and tools... prob not

**Comment by BuckarooBanzay** [comment]
what about some kind of _temporary_ immortality? it wouldn't be that op i guess.

![](https://i.giphy.com/media/iEs4yXq5rVlL2/200.gif)

**Comment by OgelGames** [comment]
> what about some kind of _temporary_ immortality?

hmm... maybe a use for golden carrots... 🤔

**Comment by Klaranth** [comment]
@BuckarooBanzay @OgelGames   @S-S-X 
What are the top three most rare items ( that won't kill you ) in-game? 
Furballs? Golden Carrots?  Can a Xeno be turned into an egg[tamed] item? 

Holding a rare item in hand makes one immune to dying?

**Comment by OgelGames** [comment]
What kind of rare? Items that can be obtained at any time, but still would be considered rare, would probably be Oranges and Strawberries, not sure about a third item...

But I dislike the idea of it being an item you hold that makes you immortal, because that defeats the purpose of armor and other survival things; I was thinking it would be something that you consume that makes you immortal for a short time.

What I thought, was adding something like an "Immortality Carrot", crafted from a golden carrot, lava orb, and maybe sugar (to make it taste nice ;P), and when you ate it, it would give you immortality for maybe 30 or 60 seconds.

Something like that would be useful for both preventing death (if you are quick enough to eat it), or recovering your bones from a deadly place.

**Comment by Klaranth** [comment]
If there is also barley in-game, can we have immortality Beer ?  I like the crafting idea.  But still, link it to a high XP level as well.

**Comment by S-S-X** [comment]
> What are the top three most rare items ( that won't kill you ) in-game?

Items that are supposed to require some work to get those (not mostly standing around afk) are mobs that you cannot breed and items that are not supposed to be duplicated.

Rare items that require player to actually do something: Bananas, strawberries, oranges, xenomorph eggs.
Bit easier to obtain but still not fully automated with afk farm (or some form of mass production): fish, jellyfish, turtle, rat, any mob you cannot breed.
Probably missing many things but I think having xenomorph egg in recipe would be very good option, maybe even some other animal egg with it or possibly allow different tiers that provide longer lasting effect (possible by using metadata for single output item).

Bananas should be added (back?) to dungeon chests. Do traders sell these nowdays, no? Related: https://github.com/pandorabox-io/pandorabox.io/issues/536

> Can a Xeno be turned into an egg[tamed] item?

Yes, and it can be used in crafting. Above few others.

> I dislike the idea of it being an item you hold that makes you immortal, because that defeats the purpose of armor and other survival things; I was thinking it would be something that you consume that makes you immortal for a short time.

Fully agree with that, expensive (in terms of no possible automation) consumable items would be best for this. It would also add first craftable rare item to game.

> What I thought, was adding something like an "Immortality Carrot", crafted from a golden carrot, lava orb, and maybe sugar (to make it taste nice ;P), and when you ate it, it would give you immortality for maybe 30 or 60 seconds.

Add no-full-automation item to recipe and it is good. Xenomorph eggs might taste nice too? Never tried one...

For XP requirement Xeno eggs would add 5k XP requirement to obtain eggs but for most players actual requirement would probably be higher because of high cost (30 mese).

**Comment by Klaranth** [comment]
> For XP requirement Xeno eggs would add 5k XP requirement to obtain eggs but for most players actual requirement would probably be higher because of high cost (30 mese).

Beyond 500K XP there isn't anything else to obtain privs-wise. Getting a chance for short-lived immortality at say 1 million XP might be interesting.  It most likely will also create a market for the items, and def. for the Xeno-eggs.

I assume there will be a counter on the screen for the user of this immortality item? 
.

**Comment by OgelGames** [comment]
> I assume there will be a counter on the screen for the user of this immortality item?

I was thinking a small particle effect would be nicer, which maybe could also be visible to other players.

Also not convinced on the XP restriction, IMO it should only be restricted by it's expensiveness, just like beacons.

**Comment by S-S-X** [comment]
> I assume there will be a counter on the screen for the user of this immortality item?

Yes I think there should be some indicator, it could be similar to what space suit has or some other on screen effect like so called vignette effect where you have some color filter frame or some other semi transparent effect around screen edges.

> I was thinking a small particle effect would be nicer, which maybe could also be visible to other players.

Particle effect would also be very good idea so that other players can see it.

**Comment by Klaranth** [comment]
> Particle effect would also be very good idea so that other players can see it.
Can it be a super saiyan glow from Dragonball Z?

**Comment by OgelGames** [comment]
Going to see if I can get a POC working within the next few days...

**Comment by Klaranth** [comment]
**Discussion in-game:**

1hittoaster 
How does one obtain a xeno (spawner, not the face-hugger egg-pod thing) egg?

AceRichman 
you can tame them like other mobs, feed them 30 or so mese blocks and catch them with a lasso

1hittoaster
re: xeno eggs.  then one should be able to craft a corium source by using an alloy furnace on a xeno egg and a block of 3.5% uranium.

AceRichman
Maybe, though modding also exists

SX 
But corium can already be crafted by building reactor and removing something after started....

**Comment by Klaranth** [comment]
Two years after the WIP label was added. Any updates yet? :)

**Comment by OgelGames** [comment]
No, but the proof of concept is still there, waiting for me to get around to it :)

![image](https://github.com/pandorabox-io/in-game/assets/48543043/eeca55fc-6dae-49e5-9863-27ec3899fdff)

**Comment by Klaranth** [comment]
Would this be something for your Silly mod to include in?

**Comment by OgelGames** [comment]
No, it's not silly.
